x-pack/plugins/siem/public/components/timeline/helpers.test.tsx

```diff
@@ -22,6 +22,14 @@ describe('Build KQL Query', () => {
     expect(cleanUpKqlQuery(kqlQuery)).toEqual('( name : Provider 1 )');
   });
 
+  test('Buld KQL query with one data provider as timestamp', () => {
```

Comment:
since `converTimestampFieldToQuery` accepts two different types of input:
```
 (value: string | number)
```
consider renaming the existing test to:

```
test('Build KQL query with one data provider as timestamp (string input)', () => {
```

and adding a variant of the test that provides numeric input. For example:

```
  test('Buld KQL query with one data provider as timestamp (numeric input)', () => {
    const dataProviders = cloneDeep(mockDataProviders.slice(0, 1));
    dataProviders[0].queryMatch.field = '@timestamp';
    dataProviders[0].queryMatch.value = 1521848183232;
    const kqlQuery = buildGlobalQuery(dataProviders);
    expect(cleanUpKqlQuery(kqlQuery)).toEqual('( @timestamp: 1521848183232 )');
  });
```